tasks/Restaurant.tex

```diff
@@ -0,0 +1,90 @@
+\section{Restaurant}
+The robots are tested in a real environment such as a real restaurant or a shopping mall. The test should be executed without a map at the start.
+There are \emph{two} robots helping clients in the restaurant at the same time.
+
+% \subsection{Focus}
+% This test focuses on online mapping, safe navigation in previously unknown environments, gesture detection, human-robot interaction, and manipulation in a real environment.
+
+\subsection{Main Goal}
+Taking two orders from the quests and serving the quests with the requested orders. Before taking the order, the robot can be guided by a \textit{Professional Waiter} (member of the TC). During delivering phase, the robot should finds its own way.
+
+\noindent\textbf{Reward:} 1000pts, 350 regular per order + 150 bonus per order
```

Comment:
I thought these were standardized at some lower number, unless it was said Restaurant was an exception.

Reply:
I am not sure what the standard is at the moment. I have seen some different numbers. We can talk about it in the meeting tomorrow.

Reply:
Yes last meeting my call was dropping in and out, so I didn’t hear the end of that discussion, but after reading through the rules it doesn’t seem standardized yet.

Reply:
Legacy from 2009-2014 was
- **Stage I** 1000pts
- **Stage II** 2500pts

Difficulty is proportional. We expect teams to score avg 1/3~1/2 available points.